pkgs/build-support/kernel/compress-firmware-xz.nix

```diff
@@ -1,8 +1,8 @@
-{ runCommand }:
+{ runCommand, lib }:
 
 firmware:
 
-runCommand "${firmware.name}-xz" {} ''
+(runCommand "${firmware.name}-xz" {} ''
```

Comment:
Shouldn't it go in the attrset here?  That's what that `runCommand` argument is there for!

Reply:
Looks like it, it wasn't clear from the `runCommand` doc:
```
Run the shell command `buildCommand' to produce a store path named
   `name'.  The attributes in `env' are added to the environment
   prior to running the command.
....
runCommand = name: env: runCommandWith
```

Reply:
Yeah, looks like that could use some improvement.